pkg/cmd/dev/ui.go

```diff
@@ -67,6 +69,7 @@ func getUIDirs(d *dev) (*UIDirectories, error) {
 	}
 
 	return &UIDirectories{
+		root:         path.Join(workspace, "./pkg/ui"),
```

Comment:
Nit: this call, and all the other calls to `path.Join` here, should actually be `filepath.Join`. Can you make the update while you're in the area?

Reply:
Oh right, good catch. Yep, will do.